docs-developer/index.rst

```diff
@@ -1,47 +1,25 @@
 Developer Guide
 ===============
 
-Start here
-----------
+Please first of all, read :ref:`our gettting started guide <getting_started>`.
+to learn   about :ref:`setting up your development environment <dev_env_intro>`,
+:ref:`a nice development workflow <workflow_intro>` and :ref:`our stack of technology <stack>`.
```

Comment:
"our technology stack"

Reply:
(otherwise I just imagine the pile of old tablets in our cabinet...)

Reply:
hahaha, will add a chapter about that? :D